webrtc.html

```diff
@@ -1862,6 +1868,39 @@
         </dl>
       </section>
     </section>
+    <section>
+      <h3>Session Negotiation Model</h3>
+      <p>Many changes to the PeerConnection state will require communication with
+        the remote side via the signaling channel. The app can be kept informed
+        as to when it needs to do signaling by listening to the
+        negotiationneeded event.</p>
+      <p>If an operation is performed on a RTCPeerConnection that requires signaling,
+        the connection will be marked as needing negotiation. Examples of such
```

Comment:
These things seem pretty hand-wavy to me right now - it's not clear there's anything that the runtime would do without app knowledge (at least presently) that would *require* a renegotiation.

Similarly for 'optional' renegotiations - I don't think we have any well-defined scenarios where we would want to drive a renegotiation without the app understanding why.

Generally, I think onNN is useful for simplifying the app's own logic about negotiation; it doesn't need to think about when to call createOffer. But I don't think we should imbue it with any more additional responsibilities than that.